Resources/doc/index.rst

```diff
@@ -0,0 +1,44 @@
+Installation
+============
+
+Step 1: Download the Bundle
+---------------------------
+
+Open a command console, enter your project directory and execute the
+following command to download the latest stable version of this bundle:
+
+```bash
+$ composer require symfony/swiftmailer-bundle "~2.3"
+```
+
+This command requires you to have Composer installed globally, as explained
+in the [installation chapter](https://getcomposer.org/doc/00-intro.md)
```

Comment:
and for the same reason, this should be:

```rst
in the `installation chapter`_ of the Composer documentation.
```

With this at the end of the file:

```rst
.. _`installation chapter`: https://getcomposer.org/doc/00-intro.md
```